common/src/main/resources/assets/voicechat/lang/pl_pl.json

```diff
@@ -6,151 +6,207 @@
-  "message.voicechat.no_group_permission": "Nie masz uprawnień, żeby dołączać do grup",
+  "message.voicechat.macos_no_mic_permission": "Brak uprawnień dostępu do Twojego mikrofonu. Otwórz Ustawienia systemowe > Prywatność i ochrona, a następnie sprawdź tam opcje dostępu",
+  "message.voicechat.macos_unsupported_launcher": "Twój launcher nie obsługuje uprawnień mikrofonu systemu macOS",
+  "message.voicechat.player_no_voicechat": "Gracz %s nie ma zainstalowanego moda %s",
+  "message.voicechat.mute.enabled": "Naciśnij, aby wyłączyć wyciszenie swojego mikrofonu.",
+  "message.voicechat.mute.disabled": "Naciśnij, aby wyciszyć swój mikrofon.",
+  "message.voicechat.mute.disabled_ptt": "Nie możesz wyciszać swojego mikrofonu, jeśli używsz funkcji „Naciśnij i mów”.",
+  "message.voicechat.disable.enabled": "Naciśnij, aby włączyć czat głosowy.",
+  "message.voicechat.disable.disabled": "Naciśnij, aby wyłączyć czat głosowy.",
+  "message.voicechat.disable.no_speaker": "Brak dostępnych głośników.",
+  "message.voicechat.hide_icons.enabled": "Naciśnij, aby pokazać wszystkie ikony czatu głosowego.",
+  "message.voicechat.hide_icons.disabled": "Naciśnij, aby ukryć wszystkie ikony czatu głosowego.",
+  "message.voicechat.show_group_hud.enabled": "Naciśnij, aby ukryć HUD czatu grupowego.",
+  "message.voicechat.show_group_hud.disabled": "Naciśnij, aby pokazać HUD czatu grupowego.",
+  "message.voicechat.recording.disabled": "Naciśnij, aby rozpocząć nagrywanie.",
+  "message.voicechat.recording.enabled": "Naciśnij, aby zatrzymać nagrywanie.",
+  "message.voicechat.no_speak_permission": "Nie masz uprawnień do mówienia",
+  "message.voicechat.no_listen_permission": "Nie masz uprawnień do słuchania czatu głosowego",
+  "message.voicechat.no_group_permission": "Nie masz uprawnień do dołączania do grup",
   "message.voicechat.search_hint": "Szukaj...",
-  "message.voicechat.search_empty": "Nie znaleziono żadnego gracza z taką nazwą",
-  "message.voicechat.other_volume": "Inne",
-  "message.voicechat.other_volume.description": "Głośność dowolnego dźwięku niewymienionego poniżej",
+  "message.voicechat.search_empty": "Nie znaleziono graczy o takiej nazwie",
+  "message.voicechat.other_volume": "Pozostałe",
+  "message.voicechat.other_volume.description": "Głośność wszystkich innych dźwięków niż te wymienione poniżej.",
   "message.voicechat.more_members": "i %s więcej...",
   "message.voicechat.no_groups": "Brak istniejących grup",
-  "message.voicechat.audio_type": "3D dźwięki: %s",
-  "message.voicechat.audio_type.normal": "Normalne",
-  "message.voicechat.audio_type.reduced": "Zredukowane",
+  "message.voicechat.audio_type": "Dźwięk 3D: %s",
+  "message.voicechat.audio_type.normal": "Normalny",
+  "message.voicechat.audio_type.reduced": "Zredukowany",
   "message.voicechat.audio_type.off": "Wył.",
-  "message.voicechat.invite_player": "Zaproś %s do swojej grupy",
+  "message.voicechat.invite_player": "Naciśnij, aby zaprosić gracza %s do swojej grupy.",
```

Comment:
If I may ask, where is the button with this tooltip located in the game?

According to the commit message and the source code, it appears to be related to the Social Interactions screen. But it seems to be a Fabric/Quilt-specific feature for some reason? Why is that?

I know that coremods are generally not well received in the Forge ecosystem (and I personally agree with this, as I remember the ~~good~~ old days of JAR modding and Risugami's Modloader), but perhaps there's a suitable (Neo)Forge API available for this?

Reply:
I am using mixins for injecting that button. I think this feature is not important enough to use mixins on Forge/NeoForge. Fabric needs a lot of mixins anyways so I only added it there. Ive never seen anyone using that button though.

Reply:
Thank you for confirming my assumptions. In this particular case, I'm not really able to test the above text in the game, but it should look just fine. It can always be adjusted later if needed.

And just for the record, maybe you could ask the NeoForged Team to add an appropriate API for your use case. I believe they shouldn't reject your request outright, especially since you have a valid use case for it. But, as you've already noted, it's a low-priority issue, so it might be something to consider if you are bored or something :)

Reply:
It's a tooltip, so space isn't really a concern.

They have an API for that, its not suited for this special purpose.